vendors/materialize/css/materialize.css

```diff
@@ -1020,11 +1020,11 @@
 }
 
 .teal.lighten-1 {
-  background-color: #424242 !important;
+  background-color: #26a69a !important;
```

Comment:
Actually... what's with the all color changes?
Will this affect anything in our current landing page?